### ISSUE TITLE
Unminified assets remain accessible

### ISSUE THREAD

**Issue by ejain** [sbt/sbt-web#65]
The .less files and unminified js files can be retrieved by anyone who cares to guess the URLs for these assets!

**Comment by strelec** [comment]
So? What's the progress like on this?

**Comment by huntc** [comment]
@pvlugter Do you know if this is still an issue?

**Comment by ejain** [comment]
@huntc it's still an issue in Play 2.3.7. The sbt-filter solution works for me, but shouldn't this be done by default?

**Comment by huntc** [comment]
@ejain If you could check that you're using the latest version of all sbt-web related plugins then that'd be great. sbt-web plugins have a release cycle that is different to Play's.

**Comment by ejain** [comment]
@huntc yes, my sbt-web plugins are all up to date. btw the `plugins.sbt` created with `activator new` references way outdated versions of the plugins...
